index.html

```diff
@@ -0,0 +1,53 @@
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<title>Apple</title>
+	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+	<link href="https://fonts.googleapis.com/css?family=Lato:300,400&display=swap" rel="stylesheet">
+	<link rel="stylesheet" type="text/css" href="css/style.css">
+</head>
+<body>
+	<nav class="top-nav">
+		<ul>
+			<li><i class="fa fa-apple" aria-hidden="true"></i></li>
+			<li><a href=""><span>Store</span></a></li>
+		 <li><a href=""><span>Mac</span></a></li>
+		 <li><a href=""><span>iPod</span></a></li>
+		 <li><a href=""><span>iPhone</span></a></li>
+		 <li><a href=""><span>iPad</span></a></li>
+		 <li><a href=""><span>iTunes</span></a></li>
+		 <li><a href=""><span>Support</span></a></li>
+
+      <li class="searchbox-con"><i class="fa fa-search"></i><input type="search" name="search" class="searchbox"></li>
+		</ul>
+	</nav>
+
+	<main>
+		<section class="top-header">
+			<div class="hero-section">
```

Comment:
Hi, you are making great progress but you did not correc this point from the previous review you got, please pass the images as backgorund images usng CSS
Each link is an <a> tag, with a <div> tag inside it.
The <div> tag inside the <a> tag should use an image as background -- [ ] The text on the original page is actually an image